utils/http/httpHelper.ts

```diff
@@ -0,0 +1,40 @@
+import { NextApiResponse } from 'next';
+import { ApiResponse, EmptyPayload, StatusMessage, StatusMessageType } from '../../types/apiResponse';
+
+export const HTTP_GET_METHOD = 'GET';
+export const HTTP_POST_METHOD = 'POST';
+export const HTTP_PUT_METHOD = 'PUT';
+export const HTTP_DELETE_METHOD = 'DELETE';
+export const HTTP_PATCH_METHOD = 'PATCH';
+export const HTTP_HEAD_METHOD = 'HEAD';
```

Comment:
Nit, but would it be better to make these enums like this? 
```typescript
export enum HttpMethod {
   GET = 'GET',
   POST = 'POST',
   ...
}
```

So that we can use it as `HttpMethod.GET` instead of `HTTP_GET_METHOD`.

Same for statuses below.